frontend/src/controllers/LoginController.ts

```diff
@@ -1,16 +1,21 @@
 import LoginService from "../services/LoginService";
-import {ILoginController} from "../models/ControllerTypes";
+import {ILoginController, ILoginService} from "../models/ControllerTypes";
+import {useContext} from "react";
+import {AuthContext} from "../context/AuthProvider";
 
 export default function LoginController(): ILoginController {
 
-    const service = LoginService();
+    const service: ILoginService = LoginService();
+    const jwtDecoded = useContext(AuthContext).jwtDecoded!
 
     return {
         login: (username: string, password: string) => {
             return service.login(username, password);
         },
         checkLoggedIn: () => {
```

Comment:
What is the return value? true for loggedIn?

Reply:
Genau